Source/Chargify.NET/Interfaces/IComponentInfo.cs

```diff
@@ -59,6 +61,32 @@ public enum ComponentType
         Unknown = -1
     }
 
+    public class ComponentTypeConverter : JsonConverter<ComponentType>
+    {
+        public override ComponentType Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            string value = reader.GetString().Replace("_", "").ToLowerInvariant();
+
+            switch (value)
+            {
+                case "metered_component":
```

Comment:
Won't the `.Replace` operation on line 68 mean that this always falls into the `default` branch?

Reply:
Good spotting!

Reply:
I mean, generally we don't actually care about this value, but to be complete i'll fix it.